src/stan/variational/advi.hpp

```diff
@@ -150,18 +150,143 @@ namespace stan {
       }
 
       /**
-       * Runs stochastic gradient ascent with Adagrad.
+       * Adaptively set hyperparameters for ADVI.
+       *
+       * @param variational variational distribution
```

Comment:
is the `variational` parameter actually modified through this method? If so, maybe mark it as both an in and out parameter in doxygen:
`@param[in, out]`